p4runtime_sh/shell.py

```diff
@@ -621,7 +624,6 @@ def msg(self):
 
     def _from_msg(self, msg):
         assert(self._action_id == msg.action_id)
-        self._params.clear()
```

Comment:
I was doing some testing on my changes to `__getitem__` on a table entry read from a switch, and it failed because self._params had no key/value pairs in it.  If you know a reason why `self._params.clear()` is important to keep here, let me know, but things seem to work better for my test case with this line removed.

Reply:
I think it was a typo and that it was meant to be `self._param_values.clear()`. Could you make that change?